Libraries/test/Amazon.Lambda.Annotations.SourceGenerators.Tests/WriterTests/CloudFormationJsonWriterTests.cs

```diff
@@ -252,7 +252,8 @@ public void DoNotModifyFunctionWithoutRequiredMetadata()
             Assert.Equal(100, rootToken["Resources"]["MethodNotCreatedFromAnnotationsPackage"]["Properties"]["Timeout"]); // unchanged
 
             var policies = rootToken["Resources"]["MethodNotCreatedFromAnnotationsPackage"]["Properties"]["Policies"] as JArray;
-            Assert.Equal(1, policies.Count);
+            Assert.NotNull(policies);
+            Assert.Single(policies);
```

Comment:
Having only `Assert.Single` will suffice. `Assert.NotNull` is redundant unless I am missing something.

Reply:
This is required to silence the static analyzer, technically it can be null but the setup ensures it will never be. 

Just another safe guard.